providers/universal-provider/src/providers/eip155.ts

```diff
@@ -159,7 +159,7 @@ class Eip155Provider implements IProvider {
             },
           ],
         },
-        chainId: args.chainId,
+        chainId: this.namespace.chains?.[0], // set request chainId to the first chain in the namespace to bypass the chainId validation
```

Comment:
Covers the "what" but think it's still missing the "why" here: 

`// Sending a previously unapproved chainId will cause namespace validation failure -> set request chainId to ...`

Is this correct?